themes/finna2/templates/search/advanced/multiselect.phtml

```diff
@@ -22,5 +22,8 @@
     <ul aria-multiselectable="true" role="listbox" aria-label="<?= $this->transEsc('Add filter') . ' ' . $this->transEsc($this->list['label']) ?>" tabindex="0" class="finna-multiselect done" aria-activedescendant=""></ul>
     <button type="button" class="finna-multiselect clear btn btn-link"><?= $this->transEsc('Clear Selections') ?></button> 
   </div>
+  <?php if (!$this->translationEmpty($field . '_facet_information_html')): ?>
+    <?=$this->translate($field . '_facet_information_html'); ?>    
```

Comment:
Tällä voisi olla wrapperina div jollakin luokalla, esim. `<div class="facet-info">` kuten sidefaceteissa.

Reply:
Pitäisikö tässä olla jotain tekstejä mukana, vai miten tätä on tarkoitus käyttää?

Reply:
Olisi kenttään liittyvä oma infoteksti käännös. Tähän myös liittyi se .additional-info luokka. Esim. tämän kyseisen tiketin havainnekuvaan liittyen: usage_rights_ext_str_mv_facet_information_text =  <div class="information additional-info"><p class="information-text">Teksti 1</p></div><div class="information additional-info"><p class="information-text">Teksti 2</p><a class="additional-info additional-info-link" href="<?=$this->url('content-page', ['page' => 'terms'])?>#terms">Linkki</a></div>
Tekstit olisivat sitä tiketin havainnekuvaa vastaavia.

Reply:
Oletko testannut tuota käytännössä? Mitä url:iin tulee osoitteeksi?

Reply:
Testasin vielä sitä linkkiä ja eihän se toimi. Menisi tällä tavalla /Search/ jälkeen eikä /Content/ ..eli käyttäisin perus href?